vg/vgtex/canvas.go

```diff
@@ -0,0 +1,272 @@
+// Copyright 2016 The gonum Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package vgtex provides a vg.Canvas implementation for LaTeX, using the pgf LaTeX package.
```

Comment:
The description should directly state that LaTeX notation used in text will be interpreted and rendered by LaTeX, because that is the primary reason for this backend.